.github/workflows/main.yml

```diff
@@ -26,7 +26,6 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '14'
-          cache: 'yarn'
```

Comment:
Why did you remove this?

Reply:
I'm guessing because ops repository still has v1.0.0 and it was still referring to that and not pulling the new changes.